redesign-bookthing/src/styles/elements/_accordion.scss

```
@@ -0,0 +1,26 @@
+button {
+    padding: 0;
+    border: none;
+    font-weight: bold;
+    background-color: transparent;
+    cursor: pointer;
+    margin: 2px 0;
+    padding: 0.5em 1em;
+    color: $off-black;
+}
+
+.js-accordion__trigger {
+
+    &active {
+        color: lighten($color: $off-black, $amount: 10%);
+    }
+
+    //  &[aria-expanded="true"] {
+    //      transition: 200ms ease 0s;
```

Try this?
```suggestion
    //      transition: color ease 200ms;
```
See [MDN on transition shorthand](https://developer.mozilla.org/en-US/docs/Web/CSS/transition)